### ISSUE TITLE
Training Japanese ASR Model

### ISSUE THREAD

**Issue by aguang1201** [SeanNaren/deepspeech.pytorch#597]
I want to train a Japanese ASR model.I have two questions:
1,There is an error in loading data error  when using mp3 file , can't use mp3 file to train, is right?
2,Would you provide the japanese labels.json?
Thank you.

**Comment by kouohhashi** [comment]
Hi, 
I have trained Japanese ASR for a while with a little bit older version of this repo with my little tweak.
I use Japanese Hiragana, which is phonogram, as labels.

Here's my labels.json.
https://github.com/kouohhashi/mytutorials/blob/master/miscellaneous/labels_ja.json

But Japanese language has 2 kinds of characters. One is phonogram called Hiragana and the other is Ideogram called Kanji which is almost equal to Chinese.

So I convert Hiragana into Hiragana and Kanji mixed with other model after converting wav into Hiragana with DeepSpeech.
It's a 2 steps architecture.

I don't know this is the right way because if you include Kanji in your labels, you don't need second step.
But at that case you need to deal with 5000+ characters.

**Comment by aguang1201** [comment]
@kouohhashi 
Thank you very much for your reminder. 
Japanese ASR is more difficult than English. 
As you said, there are some Kanji besides Hiragana. 
I guess the model you trained for converting hiragana to kanji is similar to a transformer. 
I want to try another way. Try to train an ASR model to directly include Kanji and hiragana, 
so I may need a label.json file that records all kanji and hiragana. 
Do you know where to download such a label file?
Thank you very much for your answers and label file.

**Comment by kouohhashi** [comment]
I don't have good label including both Kanji and Hiragana.
I tried some experiment with labels including some Kanji, but I could not good accuracy probably because my dataset was small at that time.

**Comment by aguang1201** [comment]
Thank you very much. In addition, you may already know this  ASR dataset, I still send you the download link on kaggle:https://www.kaggle.com/bryanpark/japanese-single-speaker-speech-dataset

**Comment by marlon-br** [comment]
@aguang1201 @kouohhashi could you please share pretrained Japanese model to test how it works?

Thanks in advance
